### PR TITLE
virtio-fs: new case of share data between host and win guest

### PR DESCRIPTION
**Pull request by xiagao**
Support sharing data between host and guest for windows guest.
ID: 1878611, 1878613, 1878615, 1878617
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
(1/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto: PASS (74.99 s)
 (2/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.always: PASS (79.13 s)
 (3/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.none: PASS (216.17 s)
 (4/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto: PASS (225.91 s)
 (5/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.always: PASS (222.24 s)
 (6/6) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.none: PASS (221.91 s)

 (1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_no_writeback: PASS (70.43 s)
 (2/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_no_writeback: PASS (223.09 s)

**Comment by xiagao** [comment]
@YongxueHong
Call for your review,thanks.

File: qemu/tests/virtio_fs_share_data.py [hunk 1]
```diff
@@ -27,58 +30,155 @@ def run(test, params, env):
...
+            session.sendline(cmd_start)
+            time.sleep(2)
+            error_context.context("Check if virtiofs service is started.",
+                                  logging.info)
+            cmd_check_virtiofs = 'wmic process where caption="virtiofs.exe" list brief'
+            s, o = session.cmd_status_output(cmd_check_virtiofs)
+            if s:
+                test.fail("Virtiofs service is failed to started,"
+                          " the return info is %s." % o)
+            # a = session.cmd("wmic logicaldisk list brief")
```
Comment (YongxueHong) [inline qemu/tests/virtio_fs_share_data.py:153]: Please delete this comment line.

**Comment by xiagao** [comment]
@YongxueHong  
Updated.
thanks.

**Comment by YongxueHong** [comment]
hi, @xiagao Please update the latest test result with the latest codes.
Thanks.

**Comment by xiagao** [comment]
@YongxueHong
 (1/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto: PASS (79.34 s)
 (2/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_no_writeback: PASS (75.97 s)
 (3/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_cache.auto: PASS (284.53 s)
 (4/4) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.virtio_fs_share_data.with_no_writeback: PASS (177.72 s)

**Pull request merged**